perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -659,6 +659,50 @@ sub body {
     return $body;
 }
 
+=head2 time_ago
+
+  Returns how long ago a problem was reported in an appropriately
+  prettified duration, depending on the duration.
+
+=cut
+
+sub time_ago {
+    my ( $self, $date ) = @_;
+    $date ||= 'confirmed';
+    my $duration = time() - $self->$date->epoch;
+    my $nearest;
+
+    if ($duration < 3600) {
```

Comment:
Could this logic be factored into `Utils::prettify_duration` so it could be called without `nearest`?